hazelcast/src/main/java/com/hazelcast/query/Predicates.java

```diff
@@ -51,15 +42,6 @@ public static Predicate instanceOf(final Class klass) {
         return new InstanceOfPredicate(klass);
     }
 
-    private static Comparable readAttribute(Map.Entry entry, String attribute) {
```

Comment:
this has been moved into a package protected class `com.hazelcast.query.impl.predicates.AttributeUtils`
I'm not exactly sure about it's purpose. It's used inside `LikePredicate` and `RegexPredicate`

There is another implementation of `readAttribute` in `AbstractPredicate`. It's quite confusing, but it might be caused by my lack of understanding.